ev3dev2/sound.py

```diff
@@ -44,15 +47,29 @@ def _make_scales(notes):
     return res
 
 
+
+def get_command_processes(command):
```

Comment:
Ah, got it; this is for the speak method which uses pipes?

Reply:
yep